pkg/application/inject/fuse/injector.go

```diff
@@ -144,221 +149,188 @@ func (s *Injector) inject(in runtime.Object, pvcName string, runtimeInfo base.Ru
 	}
 	kind := typeMeta.Kind
 	log.V(1).Info("Inject application", "namespacedName", namespacedName, "kind", kind)
-
 	pods, err := application.GetPodSpecs()
 	if err != nil {
 		return
 	}
 
 	for _, pod := range pods {
-		metaObj, err := pod.GetMetaObject()
-		if err != nil {
-			return out, err
+		for pvcName, runtimeInfo := range runtimeInfos {
```

Comment:
Is it better to check labels before injectObject?

```golang
if !utils.ServerlessEnabled(metaObj.Labels) || utils.InjectSidecarDone(metaObj.Labels) {
		log.V(1).Info("Serverless injection not enabled in pod labels, skip",
			"name", namespacedName.Name,
			"namespace", namespacedName.Namespace)
		return nil
	}
```

Reply:
Done